autogluon/inference/docker/0.4/py3/cu112/Dockerfile.gpu

```diff
@@ -31,6 +31,9 @@ RUN rm -rf /usr/local/lib/python3.8/dist-packages/gluonts/nursery/tsbench
 # Remove these to address security issues; these are not used if only python package is used
 RUN rm -rf /usr/local/lib/python3.8/dist-packages/ray/jars
 
+# Updating SageMaker to address security
+RUN pip install -U sagemaker
```

Comment:
@gradientsky will this cause problems if sagemaker package updates by a major version? Do we want to pin a max and min major version of sagemaker package?

Reply:
pinned to `sagemaker<2.113,>=2.112.2`